tests/data/included/jazz/tests/_helpers/CustomReporter1.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Jazz;
+
+use Codeception\PHPUnit\ResultPrinter\Report;
+
+final class CustomReporter1 extends Report
+{
+    public function endTest(\PHPUnit\Framework\Test $test, float $time) :void
```

Comment:
Please come up with the test which doesn't use custom reporters, this test won't work on Codeception 5.

Reply:
Any suggestion? Colors?

Reply:
What else could be asserted in output?

Reply:
Fine, you will need it for Codeception 4 only anyway :)